### PR TITLE
Dashboard: Add content to the last two flip cards

### LINKED ISSUE #331: Dashboard: Add content to the last two flip cards
**Changes**
These two flip cards dont link to anything - but the ones above do. This should change.
image

Rename the right one to `About DAMAP` and then create new routes for the both of them. The content should be essentially the same  from the wall of text that was originally on the same place in the dashboard. The original text can be taken from [this commit.](https://github.com/tuwien-csd/damap-frontend/commit/2c4e7f88678de729e9d50751305bed79b354142f)

Text marked red should go to `How to create a DMP`, the green one should go to `About DAMAP`. (the screemnshot below is from the tuwien dmp tool, not DAMAP)
image

As for the styling, be creative. No one reads the textwall, so amybe there are some useful UI components, that make it more easily digestable.

Also, currently, all flip cards look like they are on the same level of importance. it would be nice to differentiate between the "action" cards and the "info" cards. Maybe add a `?` icon to the info ones, as a starter idea. 

**Additional notes**
If you need to clarify soem things regarding the content, ask Christiane.

**Acceptance Criteria**
The two cards should link to meaningful pages, that display the original info, but in a more easily digestable format.

### PR DESCRIPTION
**Pull request by andresTabiTuwien**
## Description

Add content to the two flip cards (How to create a DMP and About DAMAP) in the home page. 

https://github.com/tuwien-csd/damap-frontend/issues/331
#### What does this PR do?

image
image

### Checks

- [ ] Summary updated
- [ ] Version view updated
- [ ] Documentation added
- [ ] Tests added
- [ ] E2e tests created
- [ ] Successfully ran e2e tests before merge

closes GH-331

### CODE REVIEW

File: libs/damap/src/lib/damap.module.ts [hunk 1]
```diff
@@ -9,11 +9,15 @@ import { TranslateModule } from '@ngx-translate/core';
...
 import { GdprComponent } from './components/gdpr/gdpr.component';
+import { DamapInfoComponent } from './components/damap-info/damap-info.component';
+import { DmpInstructionsComponent } from './components/dmp-instructions/dmp-instructions.component';
 
 export const DAMAP_ROUTES: Route[] = [
   { path: '', component: DashboardComponent },
   { path: 'dashboard', component: DashboardComponent },
   { path: 'plans', component: PlansComponent },
+  { path: 'damap', component: DamapInfoComponent },
+  { path: 'how-to-create', component: DmpInstructionsComponent },
```
Comment (rekt-hard) [inline libs/damap/src/lib/damap.module.ts]: Would it make sense to group these sites under a `info` prefix?

Comment (andresTabiTuwien) [reply libs/damap/src/lib/damap.module.ts]: Yes, I will add the prefix to them

**Comment by ValentinFutterer** [comment]
I have a few small suggestions to add:

This verbiage is outdated and should reflect that we also offer an HE template
image

Is it possible to color these points? This would make it look more aligned with the `About DAMAP` tab, whose icons are also colored in the primary color.
image

Lastly, i would suggest to make these cards visually different from the upper two in same way, to highlight that they are less important. From the top of my head, this could be color, size, a question amrk or info icon, etc...

@ChristianeStork do you have something to add?

All in all i like it a lot, it looks way more appealing that the old text wall 😄

**Comment by lpandath** [comment]
> ## Description
> 
> Add content to the two flip cards (How to create a DMP and About DAMAP) in the home page.
> 
> #331
> #### What does this PR do?
> 
> image image
> #### Breaking changes
> #### Code review focus
> #### Dependencies
> ### Checks
> 
>     * [ ]  Summary updated
> 
>     * [ ]  Version view updated
> 
>     * [ ]  Documentation added
> 
>     * [ ]  Tests added
> 
>     * [ ]  E2e tests created
> 
>     * [ ]  Successfully ran e2e tests before merge
> 
> 
> closes [GH-331](https://github.com/tuwien-csd/damap-frontend/issues/331)

Hi,

Second opinion: I think we could use Angular Material’s Expansion Panel in an accordion style. This would help us organize the content better and avoid showing too much text at once, which users often miss. With the accordion, we can add icons for each step, and it also includes a ‘Next’ step 'What is a DMP, ...', making it easier to follow.

It would make the design cleaner and more user-friendly. What do you think of this idea @ChristianeStork ?
Screenshot 2024-10-21 at 11 25 08

**Comment by andresTabiTuwien** [comment]
> > ## Description
> > Add content to the two flip cards (How to create a DMP and About DAMAP) in the home page.
> > #331
> > #### What does this PR do?
> > image image
> > #### Breaking changes
> > #### Code review focus
> > #### Dependencies
> > ### Checks
> > ```
> > * [ ]  Summary updated
> > 
> > * [ ]  Version view updated
> > 
> > * [ ]  Documentation added
> > 
> > * [ ]  Tests added
> > 
> > * [ ]  E2e tests created
> > 
> > * [ ]  Successfully ran e2e tests before merge
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > closes [GH-331](https://github.com/tuwien-csd/damap-frontend/issues/331)
> 
> Hi,
> 
> Second opinion: I think we could use Angular Material’s Expansion Panel in an accordion style. This would help us organize the content better and avoid showing too much text at once, which users often miss. With the accordion, we can add icons for each step, and it also includes a ‘Next’ step 'What is a DMP, ...', making it easier to follow.
> 
> It would make the design cleaner and more user-friendly. What do you think of this idea @ChristianeStork ? Screenshot 2024-10-21 at 11 25 08

@lpandath something like this:
image

**Comment by ChristianeStork** [comment]
Thanks, everybody, that looks great!
I love the accordion and the icons in the How to create a DMP section. 
Please add HE in the ABOUT DAMAP section as well: "Delivers DMPs for FWF and Horizon Europe projects in the funder-specific formats." Are you planning to link to damap.org in the ABOUT DAMAP text?

**Review by GeoffreyKarnbach — APPROVED** [review]
LGTM!

**Pull request merged**